packages/cron-jobs/src/jobs/integrations/indexer-manager.ts

```diff
@@ -0,0 +1,45 @@
+import { decryptSecret } from "@homarr/common";
+import { EVERY_MINUTE } from "@homarr/cron-jobs-core/expressions";
+import { db, eq } from "@homarr/db";
+import { items } from "@homarr/db/schema/sqlite";
+import { ProwlarrIntegration } from "@homarr/integrations";
+
+import { createCronJob } from "../../lib";
+
+export const indexerManagerJob = createCronJob("indexerManager", EVERY_MINUTE).withCallback(async () => {
+  const itemsForIntegration = await db.query.items.findMany({
+    where: eq(items.kind, "indexerManager"),
+    with: {
+      integrations: {
+        with: {
+          integration: {
+            with: {
+              secrets: {
+                columns: {
+                  kind: true,
+                  value: true,
+                },
+              },
+            },
+          },
+        },
+      },
+    },
+  });
+
+  for (const itemForIntegration of itemsForIntegration) {
+    const integration = itemForIntegration.integrations[0]?.integration;
```

Comment:
Check ``media-organizer.ts`` job for example for iteration. We should not take an index -> this will limit the user to one integration only. As a reminder, we want them to have multiple